### PR TITLE
feat: switch network instead of disconnect

### LINKED ISSUE #127: Ask the user to change network instead of asking him to disconnect
Capture d’écran 2023-02-09 à 16 12 24

I'm not sure that it is possible to do it with starknet react right now but it's worth checking

### PR DESCRIPTION
**Pull request by irisdv**
Close #127 

## Summary by CodeRabbit

- **New Features**
	- Introduced network switching functionality in the Navbar, allowing users to switch between blockchain networks seamlessly.
	- Added a specific check for Argent wallet types to enhance wallet identification.

- **Bug Fixes**
	- Enhanced account checks in the network validation logic to prevent unnecessary checks when no account is present.

- **Documentation**
	- Updated import statements to reflect the addition of new hooks and variables related to network switching.

### CODE REVIEW

**Review by fricoben — CHANGES_REQUESTED** [review]
## E2E Test 

On Braavos, I have this when I switch

Capture d’écran 2024-11-29 à 10 13 34

Same for Keplr ... I'm not sure they support this feature

File: utils/connectorWrapper.ts [hunk 1]
```diff
@@ -90,6 +91,15 @@ export const getLastConnected = (): Connector | null => {
   return null;
 };
 
+export const isArgentWallet = (connector?: StarknetReactConnector) => {
```
Comment (fricoben) [inline utils/connectorWrapper.ts]: Testé sur Sepolia, ne marche pas pour mobile + webwallet. 

Je propose de renommer isArgentExtension et ref :)

**Review by fricoben — CHANGES_REQUESTED** [review]
Argent mobile and webwallet not working

File: utils/connectorWrapper.ts (hunk 1 above)
Comment (irisdv) [reply utils/connectorWrapper.ts]: Ah j'ai renommé en `supportSwitchNetwork` comme ça on pourra mettre à jour la fonction dès que d'autres wallet ajoutent le support

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**